regsync.yaml

```diff
@@ -10,23 +10,6 @@ defaults:
     - application/vnd.oci.image.manifest.v1+json
     - application/vnd.oci.image.index.v1+json
 sync:
-  - source: docker.io/rancher/calico-cni
```

Comment:
We do not expect any deletions on this file. Why does it happen?

Reply:
Updated.